config/configuration_singleton.rb

```diff
@@ -96,6 +96,16 @@ def production_database_path
     # FIXME: add support/handling for DATABASE_URL
     Pathname.new(ENV["DATABASE_PATH"] || dataroot.join('production.sqlite3')).expand_path
   end
```

Comment:
I think the point of production_database_path was that:

1. `bin/setup-production` is only meant to build per user databases (the production database)
2. With a separate method to get the production database path, you could presumably write tests against `bin/setup-production` even if the Rails environment was test which I think we actually did somewhere, though I do not remember.

Or it might have just been to isolate which parts of the code where modified and thus which had to be tested.